modules/backend/layouts/auth.htm

```diff
@@ -43,12 +43,12 @@
                 /* Only run on HTTPS connections
                 * Block off Front-end Service Worker from running in the Backend allowing security injections, see GitHub #4384
                 */
-                if (location.protocol === 'https:') {
+                if ((location.protocol === 'https:') && ('serviceWorker' in navigator)) {
                     // Unregister all service workers before signing in to prevent cache issues, see github issue: #3707
                     navigator.serviceWorker.getRegistrations().then(
                         function(registrations) {
                             for (let registration of registrations) {
```

Comment:
@ayumi-cloud could you switch to ES5 style. IE11 doesn't support ES6 `for ... of loop and let` (credits to @tobias-kuendig for pointing out).